src/main/menus/app.ts

```diff
@@ -206,22 +206,35 @@ function createTemplate(config: Config) {
             role: 'close',
             accelerator: 'CmdOrCtrl+W',
         }, separatorItem, ...teams.slice(0, 9).sort((teamA, teamB) => teamA.order - teamB.order).map((team, i) => {
-            return {
+            const items = [];
+            items.push({
                 label: team.name,
-                accelerator: `CmdOrCtrl+${i + 1}`,
+                accelerator: `${process.platform === 'darwin' ? 'Cmd+Ctrl' : 'Ctrl+Shift'}+${i + 1}`,
                 click() {
                     WindowManager.switchServer(team.name);
                 },
-            };
-        }), separatorItem, {
-            label: 'Select Next Server',
+            });
+            if (WindowManager.getCurrentTeamName() === team.name) {
+                team.tabs.slice(0, 9).sort((teamA, teamB) => teamA.order - teamB.order).forEach((tab, i) => {
+                    items.push({
+                        label: `    ${tab.name}`, // TODO
```

Comment:
What is the TODO here?

Reply:
This should change to a display name once https://github.com/mattermost/desktop/pull/1680 is merged.